src/ralph/backends/database/es.py

```diff
@@ -80,21 +77,13 @@ def to_documents(self, stream, get_id):
                 "_op_type": self.op_type,
             }
             if self.op_type == "update":
-                action.update(
-                    {
-                        "doc": item,
-                    }
-                )
+                action.update({"doc": item})
             elif self.op_type in ("create", "index"):
-                action.update(
-                    {
-                        "_source": item,
-                    }
-                )
+                action.update({"_source": item})
             yield action
 
-    def put(self, chunk_size=500, ignore_errors=False):
-        """Write documents streamed from the standard input to the instance index"""
+    def put(self, stream, chunk_size=500, ignore_errors=False):
```

Comment:
And thus :point_down: 

```suggestion
    def put(
        self,
        stream: TextIO,
        chunk_size: int = 500,
        ignore_errors: bool = False,
    ):
```